### PR TITLE
[USWDS Site] - Documentation: Fix typos.

### LINKED ISSUE #857: Typo in maturity-model.md
User reported a typo on the [uswds-site/pages/documentation/maturity-model.md](https://github.com/uswds/uswds-site/blob/c37e9fdb53c93584ae3a1bbc5c5da31005579e62/pages/documentation/maturity-model.md) page

### URL / Link to page
https://designsystem.digital.gov/maturity-model/#level-2-follow-user-experience-guidance

### What seems to be the problem (1 sentence)
Under the Heading: Level 2: Following User... 
"You should follow the UX guidance **even it you** don’t use USWDS code." 

Have a look at what this page used to be like in the Wayback Machine
https://web.archive.org/web/*/digitalgov.gov

### PR DESCRIPTION
**Pull request by mejiaj**
## Description

Fixes #857. 

## Additional information
Did a quick spellcheck on other pages and fixed those too.

--- 

Before you hit Submit, make sure you’ve done whichever of these applies to you:

- [ ] Follow the [18F Front End Coding Style Guide](https://pages.18f.gov/frontend/) and [Accessibility Guide](https://pages.18f.gov/accessibility/checklist/).
- [ ] Run `npm test` and make sure the tests for the files you have changed have passed.
- [ ] Run your code through [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/) and make sure it’s error free.
- [x] Title your pull request using this format: [Website] - [UI component]: Brief statement describing what this pull request solves.

### CODE REVIEW

File: pages/about/overview.md [hunk 1]
```diff
@@ -3,5 +3,5 @@ title: About USWDS
 permalink: /about/
 layout: styleguide
 category: About USWDS
-lead: Here you can find the latest news and information about the U.S. Web Design System. Read our latest release notes, learn about USWDS’s impact in the government, and learn how we conduct user research to continuously improve our product and process.
----
\ No newline at end of file
+lead: Here you can find the latest news and information about the U.S. Web Design System. Read our latest release notes, learn about USWDS’ impact in the government, and learn how we conduct user research to continuously improve our product and process.
```
Comment (thisisdano) [inline pages/about/overview.md]: I don't agree with this `S'` change

File: pages/documentation/getting-started/designers.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ As a complement to this visual style guidance, we strongly recommend you use [18
 
 <div class="site-note"><strong>Note:</strong> USWDS 2 design assets are only currently available in Sketch and Adobe XD formats.</div>
 
-The site contains HTML mockups of common UI components designed to follow USWDS’s visual style guide. To view the specs of each design live on this website (padding, margins, stroke weight, line-height, and so on), use your browser’s developer tools.
+The site contains HTML mockups of common UI components designed to follow USWDS’ visual style guide. To view the specs of each design live on this website (padding, margins, stroke weight, line-height, and so on), use your browser’s developer tools.
```
Comment (thisisdano) [inline pages/documentation/getting-started/designers.md]: Nor this one.

File: pages/documentation/guidance/performance/why.md [hunk 3]
```diff
@@ -52,4 +52,5 @@ Measuring performance ensures the team:
 
 ## Conclusion
 
-As technology becomes more prevelant in the Unites States, it's more common that the primary way Americans interact with the government is through the web. Ensuring that government sites perform quickly for all ensures that all Americans have a good user experience and aren't blocked from the web services they require. Tracking and improving a site's performance is a first step towards ensuring that a site can reach as wide of an audience as possible.
+As technology becomes more prevalent in the Unites States, it's more common that the primary way Americans interact with the government is through the web. Ensuring that government sites perform quickly for all ensures that all Americans have a good user experience and aren't blocked from the web services they require. Tracking and improving a site's performance is a first step towards ensuring that a site can reach as wide of an audience as possible.
+s
```
Comment (thisisdano) [inline pages/documentation/guidance/performance/why.md]: What's this?

File: pages/documentation/guidance/performance/why.md [hunk 4]
```diff
@@ -34,7 +34,7 @@ A common excuse by a team  for not tracking web performance, is that the site ap
...
+The prevalence of mobile usage of websites also significantly contributes to a site's speed for a customer. According to the [Digital analytics program](https://analytics.usa.gov/), roughly 37% of traffic to government sites is from mobile devices. While wired and wi-fi networks provide relatively consistent performance, mobile networks can rapidly change from fast 4G connections, to slow 3G connections, to periods of being disconnected. Smaller devices, such as mobile phones and older devices don't have as much CPU power to quickly download and process multiple resources. By limiting the measurement of the performance of a site to just a user on a fast connection and device, a huge amount of the site’s user base could be left with a site that’s very difficult to use and interact with, due to poor performance.
```
Comment (thisisdano) [inline pages/documentation/guidance/performance/why.md]: Hm, what changed here?

File: pages/documentation/guidance/performance/why.md (hunk 3 above)
Comment (mejiaj) [reply pages/documentation/guidance/performance/why.md]: `prevelant → prevalent`

Comment (mejiaj) [reply pages/documentation/guidance/performance/why.md]: `signifigantly → significantly`

File: pages/documentation/guidance/performance/why.md (hunk 4 above)
Comment (mejiaj) [reply pages/documentation/guidance/performance/why.md]: `signifigantly → significantly`

**Pull request merged**